.gitignore

```diff
@@ -7,6 +7,9 @@ wwwbase/sitemap.xml
 wwwbase/sitemap*.xml.gz
 wwwbase/stat/sysstatgraph/data.json
 
+# Local certificate file
+cacert.pem
```

Comment:
Certificatul de la secțiunea "(Posibil) Descărcați un certificat pentru Curl" din "Instrucțiuni de instalare". Am preferat să-l țin în folderul dexonline, să nu-mi fie amestecate fișierele prin sistem.